ui/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-jessie as build-stage
+
+ARG ENABLE_LOCAL_LOGIN=YES
+ENV REACT_APP_ENABLE_LOCAL_LOGIN=$ENABLE_LOCAL_LOGIN
```

Comment:
we're checking explicitly for  ``YES`` https://github.com/inspirehep/inspirehep/blob/master/ui/src/user/index.jsx#L34